libbeat/k8skeystore/kubernetes_keystore.go

```diff
@@ -0,0 +1,146 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package k8skeystore
```

Comment:
discuss code org :)

I wonder if we really want to put this as a top-level package in libbeat.
We already have a package named `libbeat/common/kubernetes` and one named `libbeat/autodiscover/providers/kubernetes` (there might be others?). We should not spread out functionality for one service type thrughout the code base. Consider to move the package to one of the existing ones.

Reply:
👍